app/forms/mappings/AddressMappings.scala

```diff
@@ -45,6 +45,7 @@ trait AddressMappings extends Mappings {
   protected def postcodeOrCountryMapping(isUKAddress: Boolean): (String, Mapping[String]) = {
     if (isUKAddress) {
       "postCode" -> text("error.required.postcode")
+        .transform[String](_.trim,identity)
```

Comment:
You can do the same thing you have done above here and remove the `.transform` call

Reply:
Resolved